tests/genai/optimize/test_base.py

```diff
@@ -0,0 +1,97 @@
+from unittest.mock import patch
+
+import pandas as pd
+import pytest
+
+pytest.importorskip("dspy", minversion="2.6.0")
+
+from mlflow.entities.model_registry import Prompt
+from mlflow.exceptions import MlflowException
+from mlflow.genai.optimize import optimize_prompt
+from mlflow.genai.optimize.types import LLMParams, OptimizerConfig
+from mlflow.genai.scorers import scorer
+from mlflow.tracking._model_registry.fluent import register_prompt
+
+
+@pytest.fixture
+def test_prompt():
```

Comment:
```suggestion
def sample_prompt():
```

Let's rename this because this is not a test.

Reply:
This is not recognized as test case since `@pytest.fixture` is attached, but agree to rename it to avoid confusion.